src/helpers/MessageLabels/MessageLabels.js

```diff
@@ -260,6 +260,10 @@ export const messages = defineMessages({
     id: 'Show groups of users below',
     defaultMessage: 'Show groups of users below',
   },
+  UrlclipboardCopy: {
+    id: 'URL copied to clipboard',
+    defaultMessage: 'URL copied to clipboard',
+  },
```

Comment:
If this message is targeted toward users instead of developers, I would suggest that the user-facing elements use the word "Link" instead of "URL". Thoughts?

Also is `UrlclipboardCopy` the correct casing? I don't know the convention to use when the leading characters are an acronym, and I would assume that the "c" in "clipboard" should be camelCase. Thus one of these may be more correct:

- `urlClipboardCopy`
- `URLClipboardCopy`

Reply:
[de9e57f](https://github.com/plone/volto/pull/4287/commits/de9e57f3797e705a451351046a9688722a28500c)